resources/config/snemo/demonstrator/simulation/primary_events/1.3/variants/models/peg_generators.csv

```diff
@@ -0,0 +1,70 @@
+Nd150.0nubb_rhc_lambda_0 : Neutrinoless double beta decay of Nd-150, 0nubb(rhc-lambda) 0+ -> 0+ {2n} : DBD/Nd150 :
+Nd150.0nubb_rhc_lambda_0_2 : Neutrinoless double beta decay of Nd-150, 0nubb(rhc-lambda) 0+ -> 0+, 2+ {N*} : DBD/Nd150 :
+Nd150.2nubb : Two neutrino double beta decay of Nd-150, 2nubb : DBD/Nd150 :
+Nd150.0nubbM1 : Neutrinoless double beta decay of Nd-150, 0nubbM1 : DBD/Nd150 :
+Nd150.0nubbM2 : Neutrinoless double beta decay of Nd-150, 0nubbM2 : DBD/Nd150 :
+Ac228 : Ac-228 decay : Background :
+Th234 : Th-234 decay : Background :
+Tl207 : Tl-207 decay : Background :
+Tl208 : Tl-208 decay : Background :
+Bi214 : Bi-214 decay : Background :
+Bi212 : Bi-212 decay : Background :
+Bi210 : Bi-210 decay : Background :
+Bi214_Po214 : Bi-214/Po-214 decay : Background :
+Bi212_Po212 : Bi-212/Po-212 decay : Background :
+Pa234m : Pa-234m decay : Background :
+K40 : K-40 decay : Background :
+Y90 : Y-90 decay : Background :
+Sr90 : Sr-90 decay : Background :
+Pa231 : Pa-231 decay : Background :
+Eu152 : Eu-152 decay : Background :
+Eu154 : Eu-154 decay : Background :
+Pb210 : Pb-210 decay : Background :
+Pb211 : Pb-211 decay : Background :
+Pb212 : Pb-212 decay : Background :
+Pb214 : Pb-214 decay : Background :
+Ra226 : Ra-226 decay : Background :
+Co60 : Co-60 decay : Calibration :
+Bi207 : Bi-207 decay : Calibration :
+Am241 : Am-241 decay : Calibration :
+Na22 : Na-22 decay : Calibration :
+Mn54 : Mn-54 decay : Calibration :
+Cs137 : Cs-137 decay : Calibration :
+Se82.2nubb_2MeV : Two neutrino double beta decay of Se-82, 2nubb : DBD/Se82 :
+electron.20keV : Electron with monokinetic energy @ 20 keV : Miscellaneous :
+electron.50keV : Electron with monokinetic energy @ 50 keV : Miscellaneous :
+electron.100keV : Electron with monokinetic energy @ 100 keV : Miscellaneous :
+electron.200keV : Electron with monokinetic energy @ 200 keV : Miscellaneous :
+electron.500keV : Electron with monokinetic energy @ 500 keV : Miscellaneous :
+electron.1MeV : Electron with monokinetic energy @ 1 MeV : Miscellaneous :
+electron.2MeV : Electron with monokinetic energy @ 2 MeV : Miscellaneous :
+electron.3MeV : Electron with monokinetic energy @ 3 MeV : Miscellaneous :
+electron.50-2000keV_flat : Electron with energy in the [50keV-2MeV] range : Miscellaneous :
+gamma.20keV : Gamma with monokinetic energy @ 20 keV : Miscellaneous :
+gamma.50keV : Gamma with monokinetic energy @ 50 keV : Miscellaneous :
+gamma.100keV : Gamma with monokinetic energy @ 100 keV : Miscellaneous :
+gamma.500keV : Gamma with monokinetic energy @ 500 keV : Miscellaneous :
+gamma.1MeV : Gamma with monokinetic energy @ 1 MeV : Miscellaneous :
+gamma.2MeV : Gamma with monokinetic energy @ 2 MeV : Miscellaneous :
+gamma.2615keV : Gamma with monokinetic energy @ 2.615 MeV : Miscellaneous :
+versatile_generator : Particle with monokinetic energy : User : if_versatile
+flat_versatile_generator : Particle with flat energy : User : if_flat_versatile
```

Comment:
As above, if this is a generated file, it shouldn't be in version control.

Reply:
Generated files does not systematically mean "dynamically" generated. Some devel tools may be used to automate some products that still have to be considered as static. There can be very good reasons not to dynamically build some material as installation step only. My default approach for resource files is to make things static and not defer the production of such critical stuff to the build system even if it provides very fancy  functionalities.
Of course we can consider things  case by case but an unique approach is simpler for now.

Concerning this specific file, it can be safely removed. When the final "primary_events/1.3"
will be released (after more addons from MCC2 requests), we will make a new one and push/generate it in the standard documentation repo.